models/car.go

```diff
@@ -0,0 +1,46 @@
+package models
+
```

Comment:
Im starting to think that it may be worthwhile to do it like rails and have a UserChangeSet struct that is all pointers ie
```go
type CarChangeSet struct {
	Make *string
	Model *string
	Year *int
	Color *string
	UserID *string // this one you would set in adapter
} 
```